src/modules/nfts/components/wizard/PriceSummary.tsx

```diff
@@ -1,16 +1,17 @@
 import { Divider, Form, Row, Col } from 'antd';
 import React, { useContext, useEffect, useRef, useState } from 'react';
-import Image from 'next/image';
 import { StepWizardChildProps } from 'react-step-wizard';
 import styled from 'styled-components';
 import Button from '@/common/components/elements/Button';
+import SolSymbol from '@/common/components/elements/SolSymbol';
 import Paragraph from 'antd/lib/typography/Paragraph';
 import { Coingecko, Currency } from '@metaplex/js';
 import NavContainer from '@/modules/nfts/components/wizard/NavContainer';
 import { WalletContext } from '@/modules/wallet';
 import { Connection, LAMPORTS_PER_SOL, PublicKey } from '@solana/web3.js';
 import { NFTPreviewGrid } from '@/common/components/elements/NFTPreviewGrid';
 import { FilePreview } from 'pages/nfts/new';
+import { Flex } from 'antd-mobile';
```

Comment:
@sjc5 Are we using antd-mobile in many other places, or will we start doing so? :)

Reply:
hmm, I think that was an auto-import error, as I don't remember adding it.  let me double check.